conans/client/command.py

```diff
@@ -1373,6 +1374,11 @@ def get(self, *args):
         except ConanException:
             reference = args.reference
             package_id = args.package
+
+            if package_id:
+                self._user_io.out.warn("Usage of `--package` argument is deprecated."
+                                       " Use a full reference instead: "
+                                       "`conan get [...] {}:{}`".format(reference, package_id))
```

Comment:
I liked it, dont need to think, just copy and go.

Reply:
For the copy/paste I need to unroll the other arguments... maybe I should do it 🤔

Reply:
well, it would be really great for lazy people (including me) :joy_cat:

Reply:
yes, would be nice to keep it